docker/Dockerfile

```diff
@@ -14,8 +14,8 @@ FROM --platform=$build_for python:3.10.3-slim-bullseye as base
 # N.B. The refs updated automagically every release via bumpversion
 # N.B. dbt-postgres is currently found in the core codebase so a value of dbt-core@<some_version> is correct
 
-ARG dbt_core_ref=dbt-core@v1.1.0b1
-ARG dbt_postgres_ref=dbt-core@v1.1.0b1
+ARG dbt_core_ref=dbt-core@v1.1.0rc1
+ARG dbt_postgres_ref=dbt-core@v1.1.0rc1
```

Comment:
@iknox-fa should the adapter versions below be getting versioned bumped as well?

Reply:
So the versions here should simply be the most recent that we want to push out docker images for.
`dbt_core` and `dbt_postgres` will always have whatever version core currently is and that should happen automagically with bumpversion.  The adapter versions will need to match whatever the current stable version of the adapter is.  This is not currently automated.